netman/adapters/switches/juniper/base.py

```diff
@@ -425,7 +425,15 @@ def set_interface_state(self, interface_id, state):
             self.logger.info("actual setting error was {}".format(e))
             raise UnknownInterface(interface_id)
 
+    def reset_interface_state(self, interface_id):
+        update = Update()
+        update.add_interface(interface_main_update(interface_id, None))
```

Comment:
I don't see how this changes the state, is the second parameter to `interface_main_update` the state? I suggest renaming the method and/or providing kwargs to make this clearer.